src/checker.rs

```diff
@@ -301,6 +318,28 @@ mod test {
         assert!(!check(&rules, &String::from("This has a's")));
     }
 
+    #[test]
+    fn test_stem_separator_regex() {
+        let rules : Rules = Rules {
+            stem_separator_regex: "[']".to_string(),
+            disallowed_words: ["Smithsonian", "DC", "Museum"].iter().map(|s| (*s).to_string()).collect(),
+            ..Default::default()
+        };
+
+        assert!(check(&rules, &String::from("The Mall has many museums.")));
+        assert!(!check(&rules, &String::from("Smithsonian's venues are in the Mall.")));
+        assert!(!check(&rules, &String::from("Do you know Smithsonian's African American Museum's location?")));
+        assert!(!check(&rules, &String::from("Washington DC's Mall has many musums.")));
+
+        let rules : Rules = Rules {
+            disallowed_words: ["Smithsonian", "DC", "Museum"].iter().map(|s| (*s).to_string()).collect(),
+            ..Default::default()
+        };
+        assert!(check(&rules, &String::from("Smithsonian's venues are in DC's Mall - this will not be checked for stem words.")));
+
+
```

Comment:
Oh, didn't write on this before, can you please also remove those two empty lines?